testsuite/features/secondary/minkvm_guests.feature

```diff
@@ -448,9 +448,14 @@ Feature: Be able to manage KVM virtual machines via the GUI
     When I run "zypper -n mr -e --all" on "kvm_server" without error control
     And I run "zypper -n rr SUSE-Manager-Bootstrap" on "kvm_server" without error control
     And I run "systemctl stop salt-minion" on "kvm_server" without error control
+    And I run "systemctl stop venv-salt-minion" on "kvm_server" without error control
```

Comment:
We could handle this action in a new step definition, so inside the method, we identify which product version we have and we perform the correct stop of services for each case.

Reply:
Wait a minute... we already have it! :dagger: 
``` 
When(/^I stop salt-minion on "(.*?)"$/) do |minion|
```